.github/workflows/docs-check.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 22.4
-          cache: yarn
+          node-version: 22
+          cache: yarn          
```

Comment:
we shouldn't re-add the white space here, and in a few other places

Reply:
Didn't added anything manually, just reverted the earlier merged PR.

Reply:
yes, but if/when we merge this should clean up the diff